libs/forecasting/src/lib/ForecastingDashboard/ForecastingDashboard.tsx

```diff
@@ -25,6 +37,22 @@ export class ForecastingDashboard extends React.Component<
   public context: React.ContextType<typeof ModelAssessmentContext> =
     defaultModelAssessmentContext;
 
+  public constructor(props: IForecastingDashboardProps) {
+    super(props);
+
+    this.state = {
+      isTransformationCreatorVisible: false
+    };
+  }
+
+  public componentDidMount(): void {
+    const transformations = new Map<number, Map<string, Transformation>>();
+    transformations.set(
+      this.context.baseErrorCohort.cohort.getCohortID(),
+      new Map<string, Transformation>()
+    );
+  }
```

Comment:
is this componentDidMount in use? I think `transformations ` here is not used?

Reply:
I had wanted to capture the initial state and put an empty map in place for the first cohort, but it turns out that's not needed given how I wrote the `render` method. Nice catch! I'll remove the entire `componentDidMount`